domains/transfer/TransferAgreementAutomation/Worker.UnitTests/TransferAgreementsAutomationWorkerTests.cs

```diff
@@ -27,18 +27,20 @@ public async Task WhenFail_CacheIsSetToUnhealthy()
         var mockHttpMessageHandler = new MockHttpMessageHandler();
         var memoryCache = new AutomationCache();
 
+        memoryCache.Cache.Get(HealthEntries.Key).Should().BeNull("Cache should initially be empty");
```

Comment:
Is there a need to do an assert on the memocache here? 
I would argue that it testing logic in the .NET memory cache and it is out of context of this test

Reply:
Removed.